local-notifications/src/definitions.ts

```diff
@@ -155,6 +155,27 @@ export interface LocalNotificationsPlugin {
    */
   requestPermissions(): Promise<PermissionStatus>;
 
+  /**
+   * Direct user to the application settings screen to configure exact alarms.
+   *
+   * In the event that a user changes the settings from granted to denied, the application
+   * will restart and any notification scheduled with an exact alarm will be deleted.
+   *
+   * Only available on Android >= 12.
```

Comment:
The method is also available on Android < 12, this should say "Only available on Android", but then in the description mention that on Android < 12 it doesn't open the settings and just returns granted.